content/howto/geodesic/authenticate-with-leapp.md

```diff
@@ -0,0 +1,182 @@
+  - Most importantly,it should not be confused with the AWS Single Sign-On URLs
+  like these (**do not use these**)
+    - `https://signin.aws.amazon.com/saml` (this is used by your IdP admin)
+    - `https://my-sso-portal.awsapps.com/start` (this means you are using AWS SSO)
+- The AWS Identity Provider ARN assigned by AWS to your IdP. Something like
+  - `arn:aws:iam::123434211234:saml-provider/company-name`
+- Your assigned/authorized AWS IAM Role ARN. Something like
+  - `arn:aws:iam::123434211234:role/role-name`
+
+##### AWS SSO
+
+If using AWS SSO, you will need:
+
+- Your AWS SSO "start URL", also known as your "portal URL". It should be 
+very close to:
+  - `https://something.awsapps.com/start`
+
+##### AWS IAM User
+
+If you are a regular IAM User who can log into the AWS Console, you should
+log into the AWS Console while setting up Leapp. Choose "My Security Credentials" 
+from the Account drop-down menu:
+- Copy and paste your MFA ARN or Serial Number from "Assigned MFA device" 
+- Click on "Create access key" to create a new access key and copy the
+Access Key ID and Secret Access Key from the console and paste directly into 
+the appropriate fields in Leapp. 
+
+## How-To
+
+### Start Geodesic
+
+If you are a Cloud Posse client, you will have a customized version of Geodesic
+that sets an initial value for the environment variable `AWS_PROFILE`. Alternatively,
+you may have customized it yourself, or you may want to. In any case, we will 
+go with what you have. So start Geodesic, and at the command prompt, type
+
+```bash
+echo $AWS_PROFILE
+```
+
+This is the value you will give to the profile in Leapp when you configure it.
+If the output of `echo` is blank, use "default" for the profile name.
+
+### Install and Configure Leapp
+
+We are not going to duplicate (and try to keep in sync with updates to) Leapp's
+own documentation on how to install and configure Leapp. We will just give 
+you the big picture steps.
+
+- Visit the [Leapp website](https://leapp.cloud)
+- Download and install Leapp as instructed by the website
+- Follow the instructions (under "Docs") for configuring AWS
+
+Below is some guidance to the Leapp documentation that applies as of the
+time of this writing. By the time you read this it may be out of date, but
+hopefully will still be of help in guiding you through the Leapp site.
+
+##### Key Points
+
+- The "AWS Profile" setting in Leapp must match _exactly_ the value of
+`$AWS_PROFILE` you found in Geodesic in the earlier step.
+- The "AWS Region" you set in Leapp should be the AWS Region you most often
+use, as discussed [above](#aws-region-and-credentials).
+- The "Session Alias" is completely up to you: it is the name for this
+set of credentials that you will see in the UI.
+
+As of this writing, the Leapp documentation is at [https://docs.leapp.cloud/](https://docs.leapp.cloud/) and the best set of instructions to follow are the ones under a sub-menu on the left 
```

Comment:
We don't need the disclaimer. We have hundreds of links. We have tools to validate them. 

```suggestion
The Leapp documentation is at [https://docs.leapp.cloud/](https://docs.leapp.cloud/) and the best set of instructions to follow are the ones under a sub-menu on the left 
```